models/intel/person-reidentification-retail-0031/description/person-reidentification-retail-0031.md

```diff
@@ -22,13 +22,13 @@ This is a person reidentification model for a general scenario. It uses a whole
 | Source framework                  | Caffe*                                    |
 
 The cumulative matching curve (CMC) at rank-1 is accuracy denoting the possibility to locate at least one true positive in the top-1 rank.
-Mean Average Precision (mAP) is the mean across all queries’ Average Precision (AP) and AP is defined as an area under the [precision/recall](https://en.wikipedia.org/wiki/Precision_and_recall) curve.
+Mean Average Precision (mAP) is the mean across all queries Average Precision (AP). AP is defined as the area under the [precision and recall](https://en.wikipedia.org/wiki/Precision_and_recall) curve.
```

Comment:
Why did you remove the apostrophe here? It's supposed to be possessive.

Reply:
Please replicate the edit to the other `person-reidentification-retail-*` networks, as well.